specification.md

```diff
@@ -1059,6 +1061,65 @@ Note that the referenced function definition type in this case must be `rest` (d
 
 For more information about functions, reference the [Functions definitions](#Function-Definition) section.
 
+#### Using functions for HTTP Service Invocations
+
+For a few reasons, workflow designers might see necessary making straight HTTP invocations to other systems, such as:
+
+1. Target service might not expose an OpenAPI specification descriptor
+2. Target service might not implement RESTful interfaces
+3. The use case requires a simple HTTP request invocation
+
+The specification supports making HTTP invocations via a simplified interface in the [functions definition](#Function-Definition).
+
+The HTTP invocation interface can be described in the `operation` attribute of the function definition, as shown in the example:
+
+```json
+{
+"functions": [
+  {
+    "name": "queryUserById",
+    "operation": "get:/users/{id}"
```

Comment:
i think best to keep operation to use same delimiter "#" as others would do just

"METHOD#EXPRESSION"

feel free to ping me if you want and we can talk